src/authn/Token.ts

```diff
@@ -35,4 +35,8 @@ export default class Token implements authn.Token {
   static fromBytes(bytes: Uint8Array): Token {
     return new Token(authn.Token.decode(bytes))
   }
+
+  toBase64(): string {
+    return Buffer.from(this.toBytes()).toString('base64url')
```

Comment:
I'm not sure which version of base64 we want to use, this is what I'm using at the moment on the node side.

Reply:
The GRPC Gateway client implementation uses a `b64Encode` function that looks like this. I was considering using the same, since there must be an equivalent deserialization function available somewhere in the gateway code.

It looks like this:
```ts
// Base64 encoding table
const b64 = new Array(64);

// Base64 decoding table
const s64 = new Array(123);

// 65..90, 97..122, 48..57, 43, 47
for (let i = 0; i < 64;)
    s64[b64[i] = i < 26 ? i + 65 : i < 52 ? i + 71 : i < 62 ? i - 4 : i - 59 | 43] = i++;

export function b64Encode(buffer: Uint8Array, start: number, end: number): string {
	let parts: string[] = null;
  const chunk = [];
  let i = 0, // output index
    j = 0, // goto index
    t;     // temporary
  while (start < end) {
    const b = buffer[start++];
    switch (j) {
      case 0:
        chunk[i++] = b64[b >> 2];
        t = (b & 3) << 4;
        j = 1;
        break;
      case 1:
        chunk[i++] = b64[t | b >> 4];
        t = (b & 15) << 2;
        j = 2;
        break;
      case 2:
        chunk[i++] = b64[t | b >> 6];
        chunk[i++] = b64[b & 63];
        j = 0;
        break;
    }
    if (i > 8191) {
      (parts || (parts = [])).push(String.fromCharCode.apply(String, chunk));
      i = 0;
    }
  }
  if (j) {
    chunk[i++] = b64[t];
    chunk[i++] = 61;
    if (j === 1)
      chunk[i++] = 61;
  }
  if (parts) {
    if (i)
      parts.push(String.fromCharCode.apply(String, chunk.slice(0, i)));
    return parts.join("");
  }
  return String.fromCharCode.apply(String, chunk.slice(0, i));
}
```

Reply:
Not quite sure, is that standard b64? It doesn't really matter much which it is I think, http headers should be able to accommodate most of them, no? The easiest for me on the node side are any of these https://pkg.go.dev/encoding/base64#pkg-variables

Reply:
Doesn't actually have to be b64 either if you have other suggestions.

Reply:
I'll leave it as-is. I think all roads lead to the same place, given the fact that the input is a `Uint8Array` and won't contain any special characters.